themes/src/main/resources/theme/base/admin/resources/js/controllers/users.js

```diff
@@ -1703,7 +1704,23 @@ module.controller('LDAPUserStorageCtrl', function($scope, $location, Notificatio
         });
     }
 
-
+    $scope.queryAndSetLdapSupportedExtensions = function() {
+        console.log('LDAPCtrl: getLdapSupportedExtensions');
+        const PASSWORD_MODIFY_OID = '1.3.6.1.4.1.4203.1.11.1';
+
+        $http.post(
+            `${authUrl}/admin/realms/${realm.realm}/ldap-supported-extensions`,
+            initConnectionTest("getLdapSupportedExtensions", $scope.instance.config)).then(
+            (response) => {
+                Notifications.success("LDAP supported extensions successfully requested.");
+                const ldapOids = response.data;
+                if (angular.isArray(ldapOids)) {
```

Comment:
This doesn't seem to work for me. My LDAP server has only single supportedExtension "1.3.6.1.4.1.1466.20036" and despite that, when I click on "Query" button, the "Enable the LDAPv3 Password Modify Extended Operation" is incorrectly switched to ON.

Hint: You can run LDAPEmbeddedServer from your IDE, which runs embedded ApacheDS server. That one doesn't have the passwdModifyOID enabled.

Reply:
Should be fixed.